### PR TITLE
size paste and lyrics bloqued

### PR DESCRIPTION
**Pull request by titor-oopart**

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 1]
```diff
@@ -13,15 +13,10 @@
...
-import javax.swing.DefaultComboBoxModel;
-import javax.swing.JButton;
-import javax.swing.JCheckBox;
-import javax.swing.JComboBox;
-import javax.swing.JLabel;
-import javax.swing.JTextField;
-import java.awt.GridBagConstraints;
-import java.awt.GridBagLayout;
-import java.awt.Insets;
+import javax.swing.*;
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java]: *

File: src/main/java/com/fundation/logic/model/search/ImageSearch.java [hunk 2]
```diff
@@ -79,19 +80,24 @@ public List searchInPath(String path) {
...
-                            && evaluateString(height, criteriaHeight)) {
-                        List<String> metadata = MetadataCommonExtractor.getSearchListMetadata();
+                            && evaluateString(height, criteriaHeight)
+                            &&evaluateString(colorSpace,criteriaColorSpace)) {
+                        List<String> metadata = MetadataImageExtractor.getSearchListMetadata();
                         CustomizedFile matchingFile = new CustomizedFile(fileExtractor.getAbsolutePath(),
                                 fileName, fileExtension, false, false,
                                 fileSize, creationDate, accessDate,
-                                modificationDate, "MimeType", "video", metadata);
+                                modificationDate, owner, "mime type", metadata);
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/model/search/ImageSearch.java]: The String "mime type" should be a variable

File: src/main/java/com/fundation/logic/model/search/ImageSearch.java [hunk 3]
```diff
@@ -67,6 +67,8 @@ public List searchInPath(String path) {
                 } else {
                     String fileName = FileInfo.getFileDenomination(fileExtractor, "name");
                     String fileExtension = FileInfo.getFileDenomination(fileExtractor, "extension");
+                    String mimeType = FileInfo.getMimeType(fileExtractor);
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/model/search/ImageSearch.java]: mimeType should be retrieved using exiftool

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java (hunk 1 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java]: done

File: src/main/java/com/fundation/logic/model/search/ImageSearch.java (hunk 2 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/model/search/ImageSearch.java]: done

File: src/main/java/com/fundation/logic/model/search/ImageSearch.java (hunk 3 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/model/search/ImageSearch.java]: done

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 4]
```diff
@@ -56,6 +58,20 @@ public void readAll() {
         }
     }
 
+    /**
+     * This method read mime type.
+     *
+     * @return
+     */
+    public static void mimeType(String mimeType) {
+        String audioMimeType = mimeType.substring(0,9);
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: I think this should be imageMimeType

Comment (titor-oopart) [reply src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: done

**Pull request merged**